lib/views/status-label.js

```diff
@@ -17,33 +18,23 @@ export default class StatusLabel {
   }
 
   render () {
-    if (!this.properties.text) return <div />
+    const classes = ['latex-status', 'inline-block']
 
-    let classNames = [
-      'latex-status',
-      'inline-block'
-    ]
-
-    if (this.properties.type) classNames.push(`latex-${this.properties.type}`)
-    if (this.properties.onClick) classNames.push('latex-status-link')
-
-    let iconClassNames = [
-      'icon',
-      `icon-${this.properties.icon}`
-    ]
-
-    if (this.properties.spin) iconClassNames.push('latex-spin')
+    if (this.properties.busy) classes.push('is-busy')
 
     return (
-      <div className={classNames.join(' ')} onclick={this.properties.onClick}>
-        {this.properties.icon ? <div className={iconClassNames.join(' ')} /> : ''}
-        <span>{this.properties.text}</span>
+      <div className={classes.join(' ')} onclick={() => latex.log.show()}>
```

Comment:
Perhaps introduce a method along the lines of `getClassNames()` or some such. That would also circumvent the problem above, altering a constant array.